run_desktop_tests.sh

```diff
@@ -30,6 +30,7 @@ cargo build --manifest-path third_party/tock/tools/sha256sum/Cargo.toml
 echo "Checking that CTAP2 builds properly..."
 cargo check --release --target=thumbv7em-none-eabi
 cargo check --release --target=thumbv7em-none-eabi --features with_ctap1
+cargo check --release --target=thumbv7em-none-eabi --features with_ctap2_1
```

Comment:
Can you add these tests to the relevant GitHub workflows?

Reply:
Added tests in cargo_check.yml and opensk_test.yml.